src/editor/Renderer.cpp

```diff
@@ -511,14 +511,19 @@ void Renderer::saveSettings() {
 void Renderer::loadSettings() {
 
 	if (g_settings.fgdPaths.size() == 0) {
-		g_settings.fgdPaths.push_back(g_settings.gamedir + "/svencoop/sven-coop.fgd");
+		g_settings.fgdPaths.push_back("/svencoop/sven-coop.fgd");
 	}
 
 	if (g_settings.resPaths.size() == 0) {
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop/");
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop_addon/");
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop_downloads/");
-		g_settings.resPaths.push_back(g_settings.gamedir + "/svencoop_hd/");
+		g_settings.resPaths.push_back("/svencoop/");
+		g_settings.resPaths.push_back("/svencoop_addon/");
+		g_settings.resPaths.push_back("/svencoop_downloads/");
+		g_settings.resPaths.push_back("/svencoop_hd/");

```

Comment:
nitpicking: leaving the slashes out would be more consistent with the game dir (which also leaves the last slash out). I want to update the path joining logic though so will probably wait until then.